src/poly/commitment/verifier.rs

```diff
@@ -109,7 +108,7 @@ impl<C: CurveAffine> OpeningProof<C> {
         // [c] P + [c * v] U + [c] sum(L_i * u_i^2) + [c] sum(R_i * u_i^-2) + delta - [z1] G - [z1 * b] U - [z2] H
```

Comment:
Before we do anything with the `msm` object we need to do

```rust
msm.scale(C::Scalar::random());
```

This will ensure that if the existing `MSM` has `is_zero() == true` then this argument won't be able to interfere with it. It's a way of keeping the msm's linearly independent.